commands/transfers/transferList.js

```diff
@@ -0,0 +1,248 @@
+import { InteractionResponseFlags } from "discord-interactions";
+import { optionsToObject, silentResponse, updateResponse, waitingMsg, postMessage } from "../functions/helpers.js";
+import { serverRoles } from "../config/psafServerConfig.js";
+
+const logWebhook = process.env.WEBHOOK;
+
+// Command handlers
+const transferList = async ({ options, interaction_id, token, dbClient, callerId, member }) => {
+    await waitingMsg({ interaction_id, token });
+    const { player, hours, positions, buyout, extra_info } = optionsToObject(options);
+
+    const { content, ephemeral } = await dbClient(async ({ transferList, teams }) => {
+        // Check if the caller is a manager
+        if (!member.roles.includes(serverRoles.clubManagerRole)) {
+            return { content: "Only managers can list players for transfer.", ephemeral: true };
+        }
+
+        // Find the caller's team based on roles
+        const roles = member.roles.map(roleId => ({ teamRoleId: roleId }));
+        const callerTeam = await teams.findOne({ active: true, $or: roles });
+        if (!callerTeam) {
+            return { content: "You must be a manager of a team to list players for transfer.", ephemeral: true };
+        }
+
+        // Check if the player is in the caller's team
+        const playerInTeam = await teams.findOne({ id: callerTeam.id, playerIds: player });
+        if (!playerInTeam) {
+            return { content: "You can only list players from your own team.", ephemeral: true };
+        }
+
+        // Add or update the player in the transfer list
+        await transferList.updateOne(
+            { playerId: player },
+            {
+                $set: {
+                    teamId: callerTeam.id,
+                    hours,
+                    positions,
+                    buyout,
+                    extra_info: extra_info || "",
+                    listedAt: new Date()
+                }
+            },
+            { upsert: true }
+        );
+
+        const content = `Player <@${player}> has been listed for transfer.`;
+        await postMessage({ content });
+        return { content, ephemeral: false };
+    });
+
+    return updateResponse({ content, ephemeral });
```

Comment:
It will always be ephemeral anyway :)
```suggestion
    return updateResponse({ content});
```